diff -urN aa/mergeant-0.52/configure.in mergeant-0.52/configure.in
--- aa/mergeant-0.52/configure.in	2004-07-17 18:21:52.000000000 +0200
+++ mergeant-0.52/configure.in	2004-08-28 21:53:22.323604976 +0200
@@ -91,7 +91,7 @@
 GETTEXT_PACKAGE=mergeant
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE")
-ALL_LINGUAS="am az ca cs da de el en_CA en_GB es fa fr hr it ml ms nl no pl pt pt_BR ru sk sr sr@Latn sv tr uk vi zh_CN"
+ALL_LINGUAS="am az ca cs da de el en_CA en_GB es fa fr hr it ml ms nb nl pl pt pt_BR ru sk sr sr@Latn sv tr uk vi zh_CN"
 AM_GLIB_GNU_GETTEXT
 
 PKG_CHECK_MODULES(MERGEANT, libgnomedb >= 1.1.3 \
